### PR TITLE
[teedy] Add chart teedy

### PR DESCRIPTION
**Pull request by carpenike**
**Description of the change**

Adds new teedy chart (document management)

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/teedy/values.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  port:
+    port: 8080
+
+persistence:
+  data:
+    enabled: false
+    emptyDir: false
+    mountPath: /data
+
+# Enabled postgres
```
Comment (bjw-s) [inline charts/teedy/values.yaml:37]:
```suggestion
# Enables postgres
```

**Review by bjw-s — APPROVED** [review]

**Review by onedr0p — CHANGES_REQUESTED** [review]
This looks to me missing the .helmignore file

**Comment by bjw-s** [comment]
Oh snap... I just merged :|

**Comment by carpenike** [comment]
ah, need to update the template to include helmignore. I'll PR this one to add it in.

**Pull request merged**